### ISSUE TITLE
🐛 Bug ➾ Multi-word tasks collide on first word (Refactor task names to use hyphens)

### ISSUE THREAD

**Issue by psmolinsky** [ecadlabs/taqueria#1454]
Epic: #1389 

### 🚥 Status (Internal Taqueria Team Use Only)

- [x] 🔬 Investigated and Verified
- [x] ⚗️ Solution Identified and Designed 
- [ ] 🧫 Dev Implementation of Fix
- [ ] 🧪 Fix Tested and Validated
- [ ] 🏆 PR Merged

### 🆘 What happened?

If a user installs Taqueria plugins that have multi-word tasks with same first word, then they collide.

This becomes obsolete after #1159 is implemented.

### 🆘 Steps to Reproduce?

1) Install Flextesa

```
taq install @taqueria/plugin-flextesa
```

2) Install some other plugin with similar task e.g. `start foo`.
3) When you run `taq start sandbox`, then it actually executes `start foo` because the other plugin was installed as the last one.
4) Open file `.taq/config.json` and reverse the order of `plugins`
5) Now, when you run `taq start foo`, then it actually executes `start sandbox`.

### 🌮 Taqueria Version

0.22.2

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jchenche** [comment]
https://github.com/ecadlabs/taqueria/issues/1159 covers this issue. I believe it's a limitation of a command parsing tool (called Yargs) we're using if I recall. @mweichert right?

**Comment by hu3man** [comment]
@mweichert, can you provide some context or an approach here?

**Comment by hu3man** [comment]
This is a limitation of yargs. Solution identified is to refactor all tasks to use hyphens instead of spaces

**Comment by mweichert** [comment]
Duplicate of #1546
